cms/templates/js/certificate-details.underscore

```diff
@@ -16,6 +16,11 @@
             <span class="certificate-value"><%= id %></span>
         </li>
      <% } %>
+     <% if (showDetails) { %>
+        <li class="collection-description certificate-description">
+            <%= description %>
+        </li>
+      <% } %>
    </ol>
```

Comment:
@mattdrayer I have to made these changes in this PR. I know these changes are related to another ticket 
"SOL-719: Expand/Collapse arrow not appearing" But in order to write JS test for methods related to expanding/collapsing functionality, I have to fix it in this PR. Arrow is now appearing and currently i am showing "Description" in expanding mode.